yarn.lock

```diff
@@ -10281,6 +10400,15 @@ klona@^2.0.4:
   resolved "https://registry.yarnpkg.com/klona/-/klona-2.0.5.tgz#d166574d90076395d9963aa7a928fabb8d76afbc"
   integrity sha512-pJiBpiXMbt7dkzXe8Ghj/u4FfXOOa98fPW+bihOJ4SjnoijweJrNThJfd3ifXpXhREjpoF2mZVH1GfS9LV3kHQ==
 
+knex-firebird-dialect@^2.0.0:
```

Comment:
This knex dialect depends on `node-firebird-driver-native`, which depends on the native client libraries (libfb).

We want to use `node-firebird` only, because it is pure JavaScript and does not depend on client libraries.

I've filed an issue with the knex dialect repo - https://github.com/Tomas2D/knex-firebird-dialect/issues/262,

but we'll probably just have to fork it and use `node-firebird` instead